app/containers/EpisodeAdminEditor/actions.js

```diff
@@ -5,11 +5,50 @@
  */
 
 import {
-  DEFAULT_ACTION,
+  UPDATE_EPISODE_PENDING,
```

Comment:
I suggest grouping the actions, i.e. in files (`actions/episodes`) or objects (`import EPISODES from './actions'`), or just importing everything. 
Reason: Importing everything makes the import much cleaner and faster to write (agree on how to do these). The import in itself is cheap.
Grouped in files you can save 'something' on performance, while still maintaining nice, clean imports.